styles.css

```diff
@@ -0,0 +1,88 @@
+body {
+  font-family: "Roboto Condensed", sans-serif;
+  max-width: 1310px;
+  margin: auto;
+  background-color: #f4f5f6;
+}
+
+body img {
+  background-color: white;
+}
+
+.nav-top {
+  background-color: #f72210;
+}
+
+.subscribe-button {
+  background-color: #ffe400;
+  font: bold 13px/24px "Roboto Condensed", sans-serif;
+  padding: 2px 12px;
+}
+
+.badge-paul {
+  height: 1px;
+  border-top: 15px solid #f72210;
+  border-left: 13px solid #fff;
+  width: 100px;
+  top: 0;
+  right: 0;
+}
+
+.fa-facebook-official,
+.fa-twitter,
+.fa-instagram,
+.fa-tumblr,
+.fa-linkedin {
+  font-size: 1.5rem !important;
```

Comment:
- please remove all the important keywords

Reply:
Thank you @elmejdki for your feedback!
We worked on the changes and we'll request another review.